helm/templates/pod_disruption_budget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
```

Comment:
This resource needs to be wrapped in some kind of flag check or it will fail if .Values.podDisruptionBudget is not present

Reply:
agreed. came here to say the same